app/models/concerns/user_multifactor_methods.rb

```diff
@@ -53,15 +53,19 @@ def mfa_required_weak_level_enabled?
       mfa_required? && mfa_ui_only?
     end
 
-    def otp_verified?(otp)
+    def ui_otp_verified?(otp)
       otp = otp.to_s
       return true if verify_digit_otp(mfa_seed, otp)
-
       return false unless mfa_recovery_codes.include? otp
       mfa_recovery_codes.delete(otp)
       save!(validate: false)
     end
 
+    def api_otp_verified?(otp)
+      return true if verify_webauthn_otp(otp)
+      ui_otp_verified?(otp)
```

Comment:
For consistency it might be worth changing L66 to the same structure, `return true if ui_otp_verified?(otp)`.